### PR TITLE
hi

### PR DESCRIPTION
**Pull request by joergbaier**
nice interview

### CODE REVIEW

**Comment by joergbaier** [comment]
plz review
@Christopher-Shea @ahlatimer @ivymarkwell @kellyjacksonio @MrAlexLau

**Review by ivymarkwell — APPROVED** [review]
:shipit:

File: README.md [hunk 1]
```diff
@@ -1,15 +1 @@
...
-The goal of this exercise is to build a basic e-commerce checkout experience. The details of implementation are up to you, and you can use additional libraries as needed.
-
-At a minimum, the application should let the user complete the following steps:
-
-- View a list of products
-- Click a button next to a product to add it to a shopping cart
-- View the contents of the shopping cart
-- Provide a shipping address and payment information
\ No newline at end of file
+lol it public
```
Comment (Christopher-Shea) [inline README.md:1]: it's

**Review by Christopher-Shea — CHANGES_REQUESTED** [review]

**Review by MrAlexLau — APPROVED** [review]
Would be nice to have specs, but feel free to address in a follow up PR if this is blocking the sprint.

**Comment by kellyjacksonio** [comment]
LGTM

**Comment by andyhite** [comment]
Hahahah shit this was an initial commit that I apparently pushed to the wrong remote. This wasn't the final thing...if you want to see the final thing, it's at github.com/shipengine/technical-assessment

And yes it's public on purpose - with enterprise github accounts you're limited to a finite number of license seats, and we don't have enough seats to accommodate adding hundreds of candidates to the org to access a private repo. Submission is via an emailed diff (on the real assessment) so the results aren't visible to other candidates.

Anyway, nice to see all you guys in one place again 😆 I'd love to catch up with everyone.

**Pull request closed without merging**